### ISSUE TITLE
Roundtripping Headers made to List

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-transform#163]
**Describe the bug**
Roundtripping text in the contract editor when there are h1 elements with numbers at the beginning will cause the formatting to change.

```
Regular text

## 1. Heading two

## 2. Heading two again

## 3. Heading two third
```
Results in (I think):
```
Regular text

1. Heading two

---

1. Heading two again

---

1. Heading two third

---
```
With horizontal rules

**Expected behavior**
Formatting should be kept as the user sees when roundtripping.

**Screenshots**
<img width="920" alt="Screen Shot 2020-02-06 at 1 47 26 PM" src="https://user-images.githubusercontent.com/36460856/73968281-7113f500-48e7-11ea-9b8c-d96664d85b0d.png">

**Additional context**
@michael-grover any other details?

**Comment by irmerk** [comment]
Furthermore, in the editor, selecting all and deleting and starting to type again results in everything but those horizontal rules being removed. Now I cannot do anything but press enter and add more horizontal rules.

**Comment by jeromesimeon** [comment]
> Furthermore, in the editor, selecting all and deleting and starting to type again results in everything but those horizontal rules being removed. Now I cannot do anything but press enter and add more horizontal rules.

This sounds to me like a separate issue about deleting horizontal rules. Has that been tested?

**Comment by irmerk** [comment]
Good point, yes it should be separate but currently we don't have a way for the user to create the horizontal rules so I'll hold off on making that an issue.

**Comment by jeromesimeon** [comment]
> Good point, yes it should be separate but currently we don't have a way for the user to create the horizontal rules so I'll hold off on making that an issue.

Can't a user load a markdown file with an horizontal rule?

**Comment by jeromesimeon** [comment]
Fixed in #165 

```
bash-3.2$ cat test/data/headingdigit2.md 
Regular text

## 1. Heading two

## 2. Heading two again

## 3. Heading two third
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/headingdigit2.md11:12:21 PM - info: 
Regular text

1\. Heading two
----

2\. Heading two again
----

3\. Heading two third
----
```

**Comment by jeromesimeon** [comment]
I think this should be fixed with that PR. please reopen if you disagree.
